sdk/python/tests/dsl/pipeline_param_tests.py

```diff
@@ -14,7 +14,7 @@
 
 from kubernetes.client.models import V1ConfigMap, V1Container, V1EnvVar
 from kfp.dsl import PipelineParam
-from kfp.dsl._pipeline_param import _extract_pipelineparams, extract_pipelineparams_from_any
+from kfp.dsl._pipeline_param import _extract_pipelineparams, extract_pipelineparams_from_any, sanitize_k8s_name
```

Comment:
The current change won't be sufficient because there's at least another copy of `sanitize_k8s_name` definition:
https://github.com/kubeflow/pipelines/blob/2f19a26ffd205c7a3384c4c250f5dbb5150a7c4d/sdk/python/kfp/compiler/_k8s_helper.py#L20